### DISCUSSION TITLE
I can't deploy the conductor v3.4.0

### DISCUSSION THREAD

**Discussion by wpz850398226** [Netflix/conductor#2796] [Q&A] [answered]
My environment:
IDE:IntelliJ IDEA Ultimate 2020.3
JDK:11.0.13
Conductor:3.4.0
Gradle 6.8.3

**Comment by wpz850398226** [comment]
![image](https://user-images.githubusercontent.com/41596989/155961341-bdc32165-a20f-4edf-9135-884811e076ca.png)
First time:deploy without any change

**Comment by wpz850398226** [comment]
![image](https://user-images.githubusercontent.com/41596989/155961651-dde83466-e7cc-4b3a-8d40-34fece996a86.png)
Second time:deploy after change "option.release = 8" to "option.release = 11"

**Comment by wpz850398226** [comment]
![image](https://user-images.githubusercontent.com/41596989/155962224-45c3423c-719c-490e-8a59-217457a1b476.png)

My gradle setting

**Reply by aravindanr** [reply]
In your screenshot showing the command line, the version of Java used by Gradle is 1.8. Conductor 3.x requires JDK 11. A handy tool to maintain multiple JDKs in the same machine is [sdkman](www.sdkman.io).

**Comment by v1r3n** [comment, marked as answer]
Can you try from the command line?  It seems your JVM is 1.8, conductor needs JDK 11+.

**Comment by wpz850398226** [comment]
I have solved the problem,just my reasons;Thank you
